test/signup.js

```diff
@@ -77,4 +79,37 @@ export async function verifyUserOverApi(username = 'automated@test.mail', passwo
     assert.equal(user.email, testUserEmail);
     assert.equal(user.first_name, testUserFirstName);
     assert.equal(user.last_name, testUserLastName);
+} */
+
+// This is a replacement of the above function `verifyUserOverApi` which requires the access token.
+// Note: access token can only requested after being loged in.
+export async function verifyUserByLogin(driver) {
+    await login(driver, testUserEmail, testUserPassword, verificationPageTitle);
```

Comment:
But this does something completely different! The user email, first name and last name still needs to be verified. Besides, it's still possible to receive the access token by making a manual request to the API endpoint.